page-connect.php

```diff
@@ -0,0 +1,65 @@
+<div class="row col-md-12">
+	<h2 class="conf-location">Connect your ownCloud!</h2>
+	<p>Create your own private, secure “Dropbox” with an old laptop and an internet connection with <em>ownCloud Proxy</em>.</p>
+		<p>Access your files from any device, anywhere at any time, and collaborate with anyone – all in software completely within your control.</p>
+</div>
+<div class="row">
+	<div class="col-lg-4 col-md-5 text-center no-underline">
+                <!--<iframe width="100%" height="360" src="https://www.youtube.com/embed/TBD?rel=0" frameborder="0" allowfullscreen></iframe>-->
+		<a href="<?php echo get_template_directory_uri(); ?>/assets/img/misc/oC-proxy.png"><img src="<?php echo get_template_directory_uri(); ?>/assets/img/misc/oC-proxy_wee.png" class="img-responsive" alt="ownCloud Proxy" /></a>
```

Comment:
Please do use the following screenshots after our rebranding :speak_no_evil: 
![logged-in](https://cloud.githubusercontent.com/assets/878997/9550394/56ecba2c-4dab-11e5-91c3-69745f26d392.png)
![not-registered](https://cloud.githubusercontent.com/assets/878997/9550396/590be058-4dab-11e5-8172-e293c6080e43.png)